cli/pazuzu/actions/project.go

```diff
@@ -0,0 +1,295 @@
+			fmt.Println(feature)
+		}
+	}
+	return nil
+}
+
+func ProjectRemoveFeatures(c *cli.Context) error {
+	destination := c.String("directory")
+	if !c.Args().Present() {
+		return errors.New("ERROR: no features to remove")
+	}
+	features := getFeaturesList(c.Args().First())
+
+	err := utils.CheckDestination(destination)
+	if err != nil {
+		return err
+	}
+
+	pazuzufilePath := utils.GetAbsoluteFilePath(destination, pazuzu.PazuzufileName)
+	pazuzuFile, success := utils.ReadPazuzuFile(pazuzufilePath)
+	if !success {
+		return nil
+	}
+
+	newFeatures := pazuzuFile.Features
+	baseImage := pazuzuFile.Base
+
+loop:
+	for i := 0; i < len(newFeatures); i++ {
+		f1 := newFeatures[i]
+		for _, f2 := range features {
+			if f1 == f2 {
+				newFeatures = append(newFeatures[:i], newFeatures[i+1:]...)
+				i--
+				continue loop
+			}
+		}
+	}
+
+	err = generateFiles(destination, baseImage, newFeatures)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func ProjectAddFeatures(c *cli.Context) error {
+	destination := c.String("directory")
+	if !c.Args().Present() {
+		return errors.New("ERROR: no features to add")
+	}
+	features := getFeaturesList(c.Args().First())
+
+	err := utils.CheckDestination(destination)
+	if err != nil {
+		return err
+	}
+
+	pazuzufilePath := utils.GetAbsoluteFilePath(destination, pazuzu.PazuzufileName)
+	pazuzuFile, success := utils.ReadPazuzuFile(pazuzufilePath)
+	baseImage := config.GetConfig().Base
+	var currentFeatures []string
+	if success {
+		baseImage = pazuzuFile.Base
+		currentFeatures = pazuzuFile.Features
+	}
+	for _, f := range features {
+		if !isFeatureInList(currentFeatures, f) {
+			currentFeatures = append(currentFeatures, f)
+		}
+	}
+
+	err = generateFiles(destination, baseImage, currentFeatures)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func ProjectShow(c *cli.Context) error {
+	if c.NArg() != 1 {
+		return errors.New("Wrong number of arguments")
+	}
+	key := c.Args().First()
+	destination := c.String("directory")
+
+	err := utils.CheckDestination(destination)
+	if err != nil {
+		return err
+	}
+
+	pazuzufilePath := utils.GetAbsoluteFilePath(destination, pazuzu.PazuzufileName)
+	pazuzuFile, success := utils.ReadPazuzuFile(pazuzufilePath)
+	if !success {
+		return errors.New("Project doesn't have configuration yet")
+	}
+	if key == "base" {
+		fmt.Printf("%s => %s\n", key, pazuzuFile.Base)
+	}
+	return nil
+}
+
+func ProjectSet(c *cli.Context) error {
+	if c.NArg() != 2 {
+		return errors.New("Wrong number of arguments")
+	}
+	key := c.Args()[0]
+	value := c.Args()[1]
+	destination := c.String("directory")
+
+	err := utils.CheckDestination(destination)
+	if err != nil {
+		return err
+	}
+
+	var features []string
+	base := config.GetConfig().Base
+
+	pazuzufilePath := utils.GetAbsoluteFilePath(destination, pazuzu.PazuzufileName)
+	pazuzuFile, success := utils.ReadPazuzuFile(pazuzufilePath)
+
+	if success {
+		base = pazuzuFile.Base
+		features = pazuzuFile.Features
+	}
+
+	if key != "base" {
```

Comment:
What if we used `switch` here? IMHO it would be easier to add new keys this way.

```go
switch key {
case "base": 
       // ...
default: 
        return errors.New("Key is not supported")
}
```